src/Uno.UWP/Storage/Helpers/AssetsManager.wasm.cs

```diff
@@ -27,7 +27,7 @@ internal partial class AssetsManager
 		private static readonly Lazy<Task<HashSet<string>>> _assets = new Lazy<Task<HashSet<string>>>(() => GetAssets(CancellationToken.None));
 		private static readonly ConcurrentEntryManager _assetsGate = new ConcurrentEntryManager();
 
-		private static async Task<HashSet<string>> GetAssets(CancellationToken ct)
```

Comment:
I think it makes more sense to keep this private, and instead to make `_assets` an internal static property.